### PR TITLE
[SchemaRegistry] renaming from archboard feedback

### PR DESCRIPTION
**Pull request by swathipil**
addressing part of #20703

### CODE REVIEW

**Review by yunhaoling — COMMENTED** [review]
LGTM overall

File: sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py [hunk 1]
```diff
@@ -84,20 +85,20 @@ def close(self):
...
-        :param str content: String representation of the schema being registered.
-        :param serialization_type: Serialization type for the schema being registered.
-         For now Avro is the only supported serialization type by the service.
-        :type serialization_type: Union[str, SerializationType]
+        :param str schema_definition: String representation of the schema being registered.
+        :param format: Format for the schema being registered.
+         For now Avro is the only supported schema format by the service.
+        :type format: Union[str, SchemaFormat]
         :keyword content_type: The content type of the request. Default value is 'application/json'.
         :paramtype content_type: str
```
Comment (yunhaoling) [inline sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py:103]: let's pull out the docstring for `content_type` first.
I remember the service team mention that they want to reserve it for schema content type/format.

File: sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py [hunk 2]
```diff
@@ -173,18 +174,18 @@ def get_schema(self, id, **kwargs):  # pylint:disable=redefined-builtin
...
         :param str group_name: Schema group under which schema should be registered.
         :param str name: Name of schema being registered.
-        :param str content: String representation of the schema being registered.
-        :param serialization_type: Serialization type for the schema being registered.
-        :type serialization_type: Union[str, SerializationType]
+        :param str schema_definition: String representation of the schema being registered.
+        :param format: Format for the schema being registered.
+        :type format: Union[str, SchemaFormat]
         :keyword content_type: The content type of the request. Default value is 'application/json'.
         :paramtype content_type: str
```
Comment (yunhaoling) [inline sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py:190]: same here, let's pull out the content_type docstring

File: sdk/schemaregistry/azure-schemaregistry/CHANGELOG.md [hunk 3]
```diff
@@ -11,8 +11,9 @@
 - `register_schema` and `get_schema_properties` methods on sync and async `SchemaRegistryClient` now take in the following parameters in the given order:
   - `group_name`, which has been renamed from `schema_group`
   - `name`, which has been renamed from `schema_name`
-  - `content`, which has been renamed from `schema_content`
-  - `serialization_type`
+  - `schema_definition`, which has been renamed from `schema_content`
```
[the remark this answers is not in the document]
Comment (swathipil) [reply sdk/schemaregistry/azure-schemaregistry/CHANGELOG.md:14]: During the review, this was the only name that Ted wanted to prefix with `schema`. I actually mentioned this to Deya, and he said that he'll update JS to `schema_definition`.

File: sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_common/_response_handlers.py [hunk 4]
```diff
@@ -45,6 +44,6 @@ def _parse_response_schema_properties(response):
 
 def _parse_response_schema(response):
     return Schema(
-        content=response.text(),
+        schema_definition=response.text(),
```
[the remark this answers is not in the document]
Comment (swathipil) [reply sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_common/_response_handlers.py:47]: same comment as above

File: sdk/schemaregistry/azure-schemaregistry/CHANGELOG.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (yunhaoling) [reply sdk/schemaregistry/azure-schemaregistry/CHANGELOG.md:14]: still feels a weird.. but "definition" itself isn't clear enough as well..
depending on that the definition is so critical, I guess I could live with the prefix

File: sdk/schemaregistry/azure-schemaregistry/samples/async_samples/sample_code_schemaregistry_async.py [hunk 5]
```diff
@@ -27,31 +27,31 @@
...
 from azure.schemaregistry.aio import SchemaRegistryClient
-from azure.schemaregistry import SerializationType
+from azure.schemaregistry import SchemaFormat
 from azure.identity.aio import ClientSecretCredential, DefaultAzureCredential
 
 
 def create_client():
     # [START create_sr_client_async]
-    SCHEMA_REGISTRY_ENDPOINT = os.environ['SCHEMA_REGISTRY_ENDPOINT']
+    SCHEMA_REGISTRY_FQN = os.environ['SCHEMA_REGISTRY_ENDPOINT']
```
Comment (yunhaoling) [inline sdk/schemaregistry/azure-schemaregistry/samples/async_samples/sample_code_schemaregistry_async.py:36]: I think we also need to update the `os.environ['SCHEMA_REGISTRY_ENDPOINT'] to be `['SCHEMA_REGISTRY_FULLY_QUALIFED_NAMESPACE']` in the samples to stay consistent with the term we choose.

**Review by yunhaoling — APPROVED** [review]
LGTM! (only nit picky on the naming of variables (os enviorn) in samples)

File: sdk/schemaregistry/azure-schemaregistry/samples/async_samples/sample_code_schemaregistry_async.py (hunk 5 above)
Comment (swathipil) [reply sdk/schemaregistry/azure-schemaregistry/samples/async_samples/sample_code_schemaregistry_async.py:36]: yeah that makes sense, I was just lazy about it :p. Good catch!

**Pull request merged**